blueprint/core/blueprint_ReactApplicationRoot.h

```diff
@@ -148,13 +193,24 @@ namespace blueprint
             // Clear error state from previous js evals
             errorText.reset();
 
-            if (beforeBundleEval)
```

Comment:
@nick-thompson, one question around these changes. There decent chunk of duplication in the two `ReactApplicationRoot::evaluate` calls now. I've just been musing over ways to get the body of the two overrides down to a single function. I'm wondering whether `ReactApplicationRoot` could just always work with a `juce::File`. It seems this wouldn't be a big issue for callers, if for some reason direct/inline JS evaluation was required users could just call EcmascriptEngine directly. It feels like the vast majority of `ReactApplicationRoot` client code will be working with .js files. What do you think?

Reply:
Yea, I think that makes plenty of sense. If the user wants more direct interaction with the js engine, they can go straight to the ecmascript engine. Sounds good!

Reply:
Cool. File based interface only on ReactApplicationRoot it is.

Reply:
@nick-thompson, resolved here: https://github.com/nick-thompson/blueprint/pull/61/commits/399e49da36a06cb5185775dd35690f1dee55b82a

Feel free to resolve this conversation if you're happy with this change.